Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -697,7 +676,7 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
     options: [UIApplication.OpenURLOptionsKey: Any]
   ) -> Bool {
     self.applicationOpenUrlProperty.value = (application, url, options)
-    return self.facebookOpenURLReturnValue.value
+    return true
```

Comment:
In my opinion this should always be `true`. If something has caused our app to open via a URL, we're going to handle that. We would only return `false` here if we specifically know our app does not handle a particular URL. And we only know that once we've tried to find a redirect URL which is asynchronous and wouldn't work for this return value.